pkg/arvo/sys/hoon.hoon

```diff
@@ -764,6 +764,18 @@
   ?~  a  b
   [i.a $(a t.a)]
 ::
+++  snap                                               ::  replace item
+  ~/  %snap
+  |*  [a=(list) b=@ c=*]
+  ^+  a
+  (weld (scag b a) [c (slag +(b) a)])
+::
+++  sert                                                ::  insert item
+  ~/  %sert
+  |*  [a=(list) b=@ c=*]
```

Comment:
I'd be inclined to reverse the order of these arguments, so as to say "insert item at index in list", as that feels more in keeping with the argument order of `+scag` in friends. I'm curious to hear if others agree.

Reply:
`(snap [index value] list)`
`(cert [index value] list)`

Reply:
huh I'd definitely keep the current version, it mirrors `put:by` and generally lets you put the heaviest thing last.

Reply:
I prefer the current order as well.

Reply:
The build is failing because it's not picking up the changes to hoon.hoon. How do i fix that?

Reply:
I concede, the consistency with `+put:by` is a good argument, as is the point that `c` will likely be syntactically heaviest.

But `+cert` is an actively confusing name, IMO. `+into` is a borrowing suggestion that occurs to me ...

Reply:
Meant that to be "boring" ...

Reply:
I like +into